packages/gatsby/src/commands/build-javascript.ts

```diff
@@ -30,7 +30,13 @@ export const buildProductionBundle = async (
       reportWebpackWarnings(stats)
 
       if (stats.hasErrors()) {
-        return reject(stats.compilation.errors)
+        const flattenStatsErrors = (stats: webpack.Stats): Error[] => [
+          ...stats.compilation.errors,
+          ...stats.compilation.children.flatMap(child =>
```

Comment:
sure, makes good sense. i went nuts because i could have sworn i saw some core-js polyfill somewhere, but surely must have hallucinated it :)

3266529